inst/private/check_and_convert.m

```diff
@@ -56,10 +58,14 @@
       % make sure values are converted to sym
       s = structfun (@(t) check_and_convert (t){:}, s, 'UniformOutput', false);
       obj{i} = s;
-    elseif (py.isinstance(x, pyeval('int')))
+    elseif (py.isinstance(x, integer_types))
       if (py.isinstance(x, pyeval('bool')))
         error ('unexpected python bool')
       end
+      if (abs (r) > intmax ('int64'))
```

Comment:
Re: timing:
````
octave:20> tic; py.isinstance(x, pyeval('bool')); toc
Elapsed time is 0.00637603 seconds.
octave:21> tic; py.isinstance(x, pyeval('bool')); toc
Elapsed time is 0.00800395 seconds.
octave:22> tic; py.isinstance(x, pyeval('bool')); toc
Elapsed time is 0.00632501 seconds.
octave:23> 
octave:23> 
octave:23> tic; abs(double(x)) > intmax('int64'); toc
Elapsed time is 0.00253701 seconds.
octave:24> tic; abs(double(x)) > intmax('int64'); toc
Elapsed time is 0.00378704 seconds.
octave:25> tic; abs(double(x)) > intmax('int64'); toc
Elapsed time is 0.00371313 seconds.
````
So its no more expensive than the bool sanity check right above it.